src/PSSparseServerTask.cpp

```diff
@@ -576,7 +581,7 @@ void PSSparseServerTask::loop(int poll_id) {
   struct sockaddr_in cli_addr;
   socklen_t clilen = sizeof(cli_addr);
 
-  buffer.resize(10 * 1024 * 1024); // reserve 10MB upfront
+  // buffer.resize(10 * 1024 * 1024); // reserve 10MB upfront
```

Comment:
Isn't ```buffer``` used?

Reply:
I dont think so. I can't see where its used. I'll double check this more thoroughly and report back